extensions/wikia/ContributionPrototype/scripts/LoadEditor.js

```diff
@@ -0,0 +1,71 @@
+require(['jquery', 'wikia.window'],
+	function ($, window) {
+		'use strict';
+
+		$(function() {
+			console.log("CP load editor");
+
+			var plugins = [];
+			var config = {
+				body: $('<div/>'),
+				element: $('<div/>'),
+				mode: 'source'
+			}
+
+			window.WikiaEditor.create(plugins,config);
+
+		});
+
+		function init(window) {
+			console.log("CP init and do something");
+
+			loadHTML().then(injectHTMLAndLoadVendor).then(loadApp);
+
+//			var url = window.wgContributionPrototypeExternalHost + '/wiki/' + window.wgDBname + '/wiki/' + window.wgTitle;
+//			$.ajax({
+//				url: window.wgContributionPrototypeExternalHost + '/wiki/' + window.wgTitle,
+//				type: "GET",
+//				beforeSend: function(xhr) {
+//					xhr.setRequestHeader('x-wikia-community', window.wgDBname);
+//				},
+//				success: function(result) {
+//					console.log(result);
+//					$("#editarea").html(result);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ACommentedCode)